apps/explorer/src/ui/Banner.tsx

```diff
@@ -29,27 +33,69 @@ const bannerStyles = cva(
                 md: 'px-3 py-2',
                 lg: 'p-5',
             },
+            shadow: {
+                true: 'shadow-md',
+            },
+            border: {
```

Comment:
Not sure if this is necessary. I thought border is 'controlled' from variant, do we want to have toasts with no borders but being positive for example? Same for shadow I think

Reply:
@pchrysochoidis I am going off from Figma reqs here

```
Toast / Message Banner
variables controllable via code switches

1-line or 2-line
Icon ON/OFF
Color good, warning, issue, neutral_grey, neutral_white, neutral_sui_blue_light, 
Stroke ON/OFF
Shadow ON/OFF
```

Both the `border` and `shadow` are toggle-able.

Reply:
I see, I would ask @mystie711 about this just to avoid doing all the compoundVariants stuff.

Reply:
Yea, let me ping in our shared chat